app/src/main/java/uk/ac/ebi/atlas/search/metadata/CellTypeWheelDao.java

```diff
@@ -121,6 +128,23 @@ public ImmutableList<ImmutableList<String>> facetSearchCtwFields(String searchTe
                 ImmutableList.of());
     }
 
+    public ImmutableList<ImmutableList<String>> facetSearchCtwFields(String searchTerm) {
+        var facetBuilder = buildFacetOrganismToOrganismPartsToCellTypesToExperimentAccessions();
+        var queryBuilder = buildQueryForFacetOfOntologyAnnotationsWithoutNotApplicableCellTypes(searchTerm)
+                .addFacet(ORGANISMS_TERMS_KEY, facetBuilder);
+
+        return extractNestedFacetTermsOrganismsAndOrganismPartsAndCellTypesAndExperimentAccessions(queryBuilder);
+    }
+
+    public ImmutableList<ImmutableList<String>> facetSearchCtwFields(String searchTerm, String species) {
```

Comment:
Could we add `.addFilterFieldByTerm(CTW_ORGANISM, species)` conditionally if the `species` parameter is not `null`? If that works, then we could remove the other `facetSearchCtwFields` method.